src/kit/Progress.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import { ShirtSize } from 'kit/Theme';
+import Tooltip from 'kit/Tooltip';
+
+import css from './Progress.module.scss';
+
+export interface BarPart {
+  bordered?: string;
+  color: string; // css color
+  label?: string;
+  percent: number; // between 0-1
+}
+
+export interface Props {
+  inline?: boolean;
```

Comment:
not sure that `inline` is a good descriptor for this -- realizing it implies "inline vs. block" or something. 

maybe something like "decorative", "flat", "outline"?

alternatively could have individual props for setting `border-radius` and `box-shadow`

Reply:
`flat` makes sense ✅